pkg/resources/account_parameter_acceptance_test.go

```diff
@@ -131,4 +131,46 @@ func TestAcc_AccountParameter_Issue3025(t *testing.T) {
 	})
 }
 
-// TODO(next pr): add more acc tests for the remaining parameters
+func TestAcc_AccountParameter_ENFORCE_NETWORK_RULES_FOR_INTERNAL_STAGES(t *testing.T) {
+	model := model.AccountParameter("test", string(sdk.AccountParameterRequireStorageIntegrationForStageCreation), "true")
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: acc.CheckAccountParameterUnset(t, sdk.AccountParameterRequireStorageIntegrationForStageCreation),
+		Steps: []resource.TestStep{
+			{
+				Config: config.FromModel(t, model),
```

Comment:
We'll have problem here because in #3307 this method is completely removed (because it was deprecated). Instead, `FromModels` should be used. But we can do a quick follow-up after both merges.